mqemitter-redis.js

```
@@ -33,6 +34,17 @@ function MQEmitterRedis (opts) {
 
   var that = this
 
+  function onError (err) {
```

If the mqemitter is closed, you should ignore the error.